client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -8,10 +9,78 @@
     set_container_metadata,
     remove_container_metadata
 )
-from ayon_core.hosts.substancepainter.api.lib import prompt_new_file_with_mesh
 
 import substance_painter.project
-import qargparse
+
+
+def _convert(substance_attr):
+    """Function to convert substance C++ objects to python instance.
+    It is made to avoid any possible ValueError when C++ objects is
+    converted to the Substance Painter Python API equivalent objects.
+
+    Args:
+        subst_attr (str): Substance attributes
+
+    Raises:
+        ValueError: Raise Error when unsupported Substance
+            Project was detected
+
+    Returns:
+        python instance: converted python instance of the C++ objects.
```

Comment:
It's not related to C++ objects, only string to python object conversions. (The Substance Painter Python API attribute could be a C++ shim but I'd say we don't care about that, but only really about that it matches the python api)

Other changes:
- According to [this example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) `Raises` goes below `Returns` in the docstring.
- Docstring argument `subst_attr` -> `substance_attr`

```suggestion
def _convert(substance_attr):
    """Return Substance Painter Python API Project attribute from string.
    
    This converts a string like "ProjectWorkflow.Default" to for example
    the Substance Painter Python API equivalent object, like:
        `substance_painter.project.ProjectWorkflow.Default`

    Args:
        substance_attr (str): The `substance_painter.project` attribute,
            for example "ProjectWorkflow.Default"

    Returns:
        Any: Substance Python API object of the project attribute.

    Raises:
        ValueError: If attribute does not exist on the
            `substance_painter.project` python api.
```